src/components/ReactP5WrapperGuard.tsx

```diff
@@ -3,23 +3,48 @@ import * as React from "react";
 import { type P5WrapperProps } from "../contracts/P5WrapperProps";
 import { type P5WrapperPropsWithSketch } from "../contracts/P5WrapperPropsWithSketch";
 import { type SketchProps } from "../contracts/SketchProps";
-import { ReactP5WrapperWithSketch } from "./ReactP5WrapperWithSketch";
+import { logErrorBoundaryError } from "../utils/logErrorBoundaryError";
 
-export function ReactP5WrapperGuard<Props extends SketchProps = SketchProps>(
+const ReactP5WrapperWithSketch = React.lazy(
+  () => import("./ReactP5WrapperWithSketch")
+);
+
+const ErrorBoundary = React.lazy(() =>
+  import("react-error-boundary").then(m => ({
+    default: m.ErrorBoundary
+  }))
+);
+
+export default function ReactP5WrapperGuard<Props extends SketchProps>(
```

Comment:
@jamesrweb I have reinstalled the packages in my demo application. There seems to be some runtime errors related to the changes in `ReactP5WrapperGuard`:

<img width="1438" alt="image" src="https://github.com/P5-wrapper/react/assets/13007925/7726fad6-a715-4d4a-b470-47e8cc898aaa">

Reply:
@jamesrweb After installing react@beta, there are still runtime errors related to `useRef`:

<img width="1427" alt="image" src="https://github.com/P5-wrapper/react/assets/13007925/72fd7d70-8074-464e-8a56-5075545d94b7">

I assume this is not related to changes in react 19 support, but it may be relevant to the `ReactP5WrapperGuard` component as I mentioned above.

It would be nice if you tested your branch with an external React app, otherwise you can release as is, but there is a good chance the package won't work I think. So it's up to you.